.github/workflows/main.yml

```diff
@@ -38,59 +38,124 @@ jobs:
+          python -m pytest -v tests/python/contrib/test_rpc_server_device.py
 
   Windows:
     runs-on: windows-2019
     steps:
-    - uses: actions/checkout@v2
-      with:
-        submodules: 'recursive'
-    - name: Set up environment
-      uses: ./.github/actions/setup
-    - name: Conda Build
-      shell: cmd /C call {0}
-      run: >-
-        conda build --output-folder=conda/pkg conda/recipe &&
-        conda install tvm -c ./conda/pkg
-    - name: Test
-      shell: cmd /C call {0}
-      run: >-
-        python -m pytest -v tests/python/all-platform-minimal-test
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Set up environment
+        uses: ./.github/actions/setup
+      - name: Conda Build
+        shell: cmd /C call {0}
+        run: >-
+          conda build --output-folder=conda/pkg conda/recipe &&
+          conda install tvm -c ./conda/pkg
+      - name: Test
+        shell: cmd /C call {0}
+        run: >-
+          python -m pytest -v tests/python/all-platform-minimal-test
+
+  Android:
+    runs-on: Ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Set up environment
+        uses: ./.github/actions/setup
+      - name: Set up java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - name: Build TVM
+        shell: bash -l {0}
+        run: |
+          mkdir build
+          cd build
+          ../tests/scripts/task_config_build_jvm.sh .
+          cmake ..
+          make
+      - name: Build TVM4J
+        run: |
+          make jvmpkg
+      - name: Build android_rpc
+        working-directory: apps/android_rpc
+        run: |
+          export PATH="${ANDROID_NDK_HOME}:$PATH"
+          gradle clean build
+      - name: Upload android_rpc APK
+        uses: actions/upload-artifact@v2
+        with:
+          name: android_rpc-debug.apk
+          path: ./apps/android_rpc/app/build/outputs/apk/debug/app-debug.apk
+      - name: Build android_deploy
+        working-directory: apps/android_deploy
+        run: |
+          export PATH="${ANDROID_NDK_HOME}:$PATH"
+          gradle clean build
+      - name: Upload android_deploy APK
+        uses: actions/upload-artifact@v2
+        with:
+          name: android_deploy-debug.apk
+          path: ./apps/android_deploy/app/build/outputs/apk/debug/app-debug.apk
+      - name: Build android_camera
+        working-directory: apps/android_camera
+        run: |
+          mkdir -p app/src/main/assets/models/
+          export TVM_NDK_CC=${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android30-clang++
+          export TVM_HOME=~/work/tvm/tvm
+          export PYTHONPATH=$TVM_HOME/python:${PYTHONPATH}
+          python3 ../../python/gen_requirements.py
+          pip3 install -r ../../python/requirements/core.txt
```

Comment:
It is better to replace relative paths through the environment variable `TVM_HOME`.

Reply:
Done